imod/mf6/model.py

```diff
@@ -235,7 +243,12 @@ def validate(self, model_name: str = "") -> StatusInfoBase:
                 bottom=bottom,
             )
             if len(pkg_errors) > 0:
-                model_status_info.add(pkg_errors_to_status_info(pkg_name, pkg_errors))
+                suggest_cleanup = False
+                if pkg_has_cleanup(pkg):
+                    suggest_cleanup = True
```

Comment:
Instead of passing on a boolean to the `pkg_errors_to_status_info` you can make it more generic by passing directly a footer to it. That way the logic of indentifying packages that can be cleaned up is only used in 1 place, this place

```
if len(pkg_errors) > 0:
  footer = SUGGESTION_TEXT if pkg_has_cleanup(pkg) else None
  model_status_info.add(pkg_errors_to_status_info(pkg_name, pkg_errors, footer)
```

Reply:
Done, thanks!